web/pgadmin/browser/server_groups/servers/databases/schemas/utils.py

```diff
@@ -192,6 +192,25 @@ def get_types(self, conn, condition, add_serials=False, schema_oid=''):
 
         return True, res
 
+    def get_geometry_types(self, conn):
+        types = []
+        sql = "SELECT * FROM pg_extension WHERE extname = 'postgis';"
```

Comment:
You've checked for count everywhere but here it is diff. Probably create a function to check extension exists and pass the conn object and extname as args and re-use it.

Reply:
Okay.